examples/basic.html

```diff
@@ -15,6 +15,12 @@
         editors: [{
           name: "Your Name",
           url: "https://your-site.com",
+          note: "blah"
```

Comment:
Once this is all working, please revert changes to basic.html

Reply:
Would you like to include `retiredDate` example in `basic.html`?

Reply:
Probably not. But we need to remember to add it to the wiki when we are ready to land this.